frontend/android/app/src/main/AndroidManifest.xml

```diff
@@ -1,49 +1,82 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
reformated this file with 2 spaces, since we use this scheme also for other xml files